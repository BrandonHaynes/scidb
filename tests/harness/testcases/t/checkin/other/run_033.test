--setup
CREATE ARRAY dense <a: double > [x=0:5,3,0, y=0:5,3,0]
load(dense,'${TEST_DATA_DIR}/dense.txt')

--test
apply(dense, sqrt, pow(a, 0.5))

--cleanup
remove (dense)

