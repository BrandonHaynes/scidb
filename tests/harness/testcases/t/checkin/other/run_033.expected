Query was executed successfully

{x,y} a
{0,0} 11
{0,1} 12
{0,2} 13
{1,0} 21
{1,1} 22
{1,2} 23
{2,0} 31
{2,1} 32
{2,2} 33
{0,3} 14
{0,4} 15
{0,5} 16
{1,3} 24
{1,4} 25
{1,5} 26
{2,3} 34
{2,4} 35
{2,5} 36
{3,0} 41
{3,1} 42
{3,2} 43
{4,0} 51
{4,1} 52
{4,2} 53
{5,0} 61
{5,1} 62
{5,2} 63
{3,3} 44
{3,4} 45
{3,5} 46
{4,3} 54
{4,4} 55
{4,5} 56
{5,3} 64
{5,4} 65
{5,5} 66

{x,y} a,sqrt
{0,0} 11,3.31662
{0,1} 12,3.4641
{0,2} 13,3.60555
{1,0} 21,4.58258
{1,1} 22,4.69042
{1,2} 23,4.79583
{2,0} 31,5.56776
{2,1} 32,5.65685
{2,2} 33,5.74456
{0,3} 14,3.74166
{0,4} 15,3.87298
{0,5} 16,4
{1,3} 24,4.89898
{1,4} 25,5
{1,5} 26,5.09902
{2,3} 34,5.83095
{2,4} 35,5.91608
{2,5} 36,6
{3,0} 41,6.40312
{3,1} 42,6.48074
{3,2} 43,6.55744
{4,0} 51,7.14143
{4,1} 52,7.2111
{4,2} 53,7.28011
{5,0} 61,7.81025
{5,1} 62,7.87401
{5,2} 63,7.93725
{3,3} 44,6.63325
{3,4} 45,6.7082
{3,5} 46,6.78233
{4,3} 54,7.34847
{4,4} 55,7.4162
{4,5} 56,7.48331
{5,3} 64,8
{5,4} 65,8.06226
{5,5} 66,8.12404

Query was executed successfully

